umake/frameworks/android.py

```diff
@@ -71,6 +71,8 @@ def parse_download_link(self, tag, line, in_download):
 
         if url is None and sha1sum is None:
             return (None, in_download)
+        if url and url.startswith("//"):
```

Comment:
This is needed because the line with the url is:
`href="//dl.google.com/android/android-sdk_r24.4.1-linux.tgz">android-sdk_r24.4.1-linux.tgz</a>`